app/src/main/java/com/hippo/ehviewer/widget/SeekBarPanel.java

```diff
@@ -54,8 +55,9 @@ public boolean onTouchEvent(@NonNull MotionEvent event) {
         if (mSeekBar == null) {
             return super.onTouchEvent(event);
         } else {
-            final float offsetX = -mSeekBar.getLeft();
-            final float offsetY = -mSeekBar.getTop();
+            View containingView = (View) mSeekBar.getParent();
```

Comment:
Oh, I forgot the parent-view of SeekBar was not SeekBarPanel.
`ViewUtils.getLocationInAncestor()` might be better in case the layout changes.

Reply:
Like this:
```java
ViewUtils.getLocationInAncestor(mSeekBar, mLocation, this);
final float offsetX = -mLocation[0];
final float offsetY = -mLocation[0];
```

Reply:
Thanks. I'll check.